Dockerfile

```
@@ -18,6 +19,10 @@ RUN apk update && apk upgrade
 # Ran separately from 'make' to avoid re-doing
 RUN git submodule update --init --recursive
 
+RUN if [ "$HEAPTRACK_BUILD" = "1" ]; then \
```

```suggestion
RUN if [ "$HEAPTRACK_BUILD" ]; then \
```

See https://github.com/waku-org/nwaku/pull/3522/files#r2227311344